rust/c509-certificate/src/cert_tbs.rs

```diff
@@ -184,9 +184,9 @@ impl Decode<'_, ()> for TbsCert {
 // ------------------Test----------------------
 
 // Notes
-// - Test is modified to match the current encode and decode where `subject_public_key`
-//   doesn't support
-// special case for rsaEncryption and id-ecPublicKey.
+// - Test is modified to match the current encode and decode where this crate
+// not yet supported `subject_public_key` special case for rsaEncryption and
+// id-ecPublicKey.
```

Comment:
The test has been modified to align with the current encoding and decoding functionality, as this crate does not yet support `subject_public_key` special case for rsaEncryption and id-ecPublicKey.